myapp/app/helpers/users_helper.rb

```diff
@@ -0,0 +1,2 @@
+module UsersHelper
+end
```

Comment:
If we dont have any function inside the module, its better not to create the class

Reply:
Removed it!
Please refer to e8e1e02